syntax = "proto3";

package NGenerativeBoltalka.Proto;

message TScoringSegment {
    repeated float Scores = 1;
}

message TScoringResponse {
    reserved 1;
    repeated TScoringSegment Segments = 2;
}
